.circleci/config.yml

```diff
@@ -47,6 +48,19 @@ workflows:
           requires:
             - check_fmt
             - test_and_coverage
+      - codacy/shell:
+          name: docker_publish_local
+          cmd: |
+            docker build -t codacy-clang-tidy .
+            docker save --output docker-image.tar $CIRCLE_PROJECT_REPONAME:latest
```

Comment:
We're not publishing the docker image to do docker hub

Reply:
Right. I add it.

Reply:
Done ✅